utils/formatting/hex.go

```diff
@@ -53,7 +53,8 @@ func (h *Hex) FromString(str string) error {
 
 // String ...
 func (h Hex) String() string {
```

Comment:
Do you think it makes sense to have the hex String() function have the same type as cb58 String() even though that part of the interface isn't currently shared?

Reply:
I think this is ok...if we change this to return an error then we can't do this, for example:

`fmt.Printf("the hex represenation is %s", [hex struct goes here])`